app/dashboard/src/components/dashboard/column/SharedWithColumn.tsx

```diff
@@ -45,21 +45,16 @@ export default function SharedWithColumn(props: SharedWithColumnPropsInternal) {
   const { item, setItem, state, isReadonly = false } = props
   const { category, setQuery } = state
   const asset = item.item
-  const { user } = authProvider.useNonPartialUserSession()
+  const { user } = authProvider.useFullUserSession()
   const dispatchAssetEvent = eventListProvider.useDispatchAssetEvent()
-
```

Comment:
Can we please keep semantic delimiters in place?

Having blank lines improve readability and group similar things together

Reply:
sure, but how do we determine when we need a delimiter? do we need one between `isFeatureUnderPaywall` and its usage (as was previously the case)?

Reply:
> sure, but how do we determine when we need a delimiter? do we need one between isFeatureUnderPaywall and its usage (as was previously the case)?

Subjectively. Listen to your heart 😊